### PR TITLE
[Fix-16770] [Workflow State Statistics] Fix workflow state statistics status shows error

### LINKED ISSUE #16770: [Bug] [Workflow State Statistics] Status shows error
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

image
image

### What you expected to happen

Display is normal

### How to reproduce

Above image

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by sdhzwc**
## Purpose of the pull request
close #16770 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-ui/src/locales/zh_CN/home.ts [hunk 1]
```diff
@@ -39,5 +39,6 @@ export default {
   dispatch: '派发',
   ready_block: '准备阻断',
   block: '阻断',
-  wait_to_run: '等待执行'
+  wait_to_run: '等待执行',
+  failover: '故障转移'
```
Comment (SbloodyS) [inline dolphinscheduler-ui/src/locales/zh_CN/home.ts]:
```suggestion
  failover: '恢复容错'
```

Comment (sdhzwc) [reply dolphinscheduler-ui/src/locales/zh_CN/home.ts]: > 

done

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by ruanwenjun — COMMENTED** [review]
The type `WAIT_TO_RUN` is never used, it would be better if you help to optimize this.

**Comment by sdhzwc** [comment]
> The type `WAIT_TO_RUN` is never used, it would be better if you help to optimize this.

The type `WAIT_TO_RUN` is never used, It can be removed, but this PR is to fix the display problem. The two do not affect each other.

**Comment by ruanwenjun** [comment]
> > The type `WAIT_TO_RUN` is never used, it would be better if you help to optimize this.
> 
> The type `WAIT_TO_RUN` is never used, It can be removed, but this PR is to fix the display problem. The two do not affect each other.

Can you fix in this PR?

**Review by ruanwenjun — APPROVED** [review]

**Pull request merged**